app/models/order.rb

```diff
@@ -0,0 +1,9 @@
+# app/models/order.rb
+
+class Order < ApplicationRecord
+  belongs_to :customer
+
+  validates :product_name, presence: true
+  validates :product_count, presence: true, numericality: { only_integer: true, greater_than: 0, less_than_or_equal_to: 100 }
```

Comment:
Looks like you chose to limit the customer to ordering a max of 100 items? That's smart to check that it's a positive integer though! 👍